Sources/ProjectDescription/Platform.swift

```diff
@@ -15,3 +15,19 @@ public enum Platform: String, Codable, Equatable, CaseIterable {
     /// The visionOS platform
     case visionOS = "visionos"
 }
+
+/// A supported platform representation.
+public enum PackagePlatform: String, Codable, Equatable, CaseIterable {
```

Comment:
This was introduced because SPM includes catalyst as a platform while many other parts of our system map catalyst to iOS.  This was a necessary distinction when inferring destinations from a SPM package.